R/chop.R

```diff
@@ -89,7 +96,13 @@ chop <- function(data, cols) {
 
 #' @export
 #' @rdname chop
-unchop <- function(data, cols, keep_empty = FALSE, ptype = NULL) {
+unchop <- function(data, cols, keep_empty = FALSE, ptype = NULL, ...) {
```

Comment:
In general, I think it's better to put the dots before the first optional argument. It's possible that a few people might have relied on partial or position based matching, but `check_dots_used()` will catch that.

Reply:
https://design.tidyverse.org/dots-position.html

Reply:
In `build_wider_spec()` I put the dots after `names_from` and `values_from` as they felt like main arguments and that's also how they were used in the test *names_glue affects output names*